test/unit/init/WorkerManager.test.ts

```diff
@@ -0,0 +1,15 @@
+import { ClusterManager, WorkerManager } from '../../../src';
+
+describe('A WorkerManager', (): void => {
+  it('can be created from a ClusterManager.', (): void => {
+    expect((): WorkerManager => new WorkerManager(new ClusterManager(4))).toBeDefined();
```

Comment:
`ClusterManager` should be mocked since this is a unit test for `WorkerManager`.